packages/runtime/test/src/interpreter/opcode-list.ts

```diff
@@ -7283,4 +7284,51 @@ describe("Teal Opcodes", function () {
 			assert.equal(1900, op.execute(stack));
 		});
 	});
+
+	describe("json_ref", function () {
+		const stack = new Stack<StackElem>();
+		const jsonByte = "{\"key0\": 0,\"key1\": \"algo\",\"key2\":{\"key3\": \"teal\", \"key4\": {\"key40\": 10}}, \"key5\": 18446744073709551615 }";
+
+		it("should return correct JSONUint64", function () {
```

Comment:
```suggestion
		it("Should return correct JSONUint64", function () {
```
and lets change it in all the cases